common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationSettings.java

```diff
@@ -315,7 +317,7 @@ public boolean getUseBroker() {
      *
      * @param useBroker True to use broker
      */
-    @SuppressFBWarnings("ME_ENUM_FIELD_SETTER")
+    @SuppressFBWarnings(ME_ENUM_FIELD_SETTER)
```

Comment:
Not related to this PR, 
but Ideally we should be ading justifications for these supression to specify why we are supressing these, @SuppressFBWarnings(value = "", justification = ""). 
or if applicable  remove the warning globally